actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Add SameSite to cookies.
+
+    `SameSite` cookie is an addition to the CSRF protection.
+    For now, the default is `:no_protection` which makes it backward compatible.
+    By switching from `:no_protection` to `:lax`
+    in `config/initializers/cookies`, the cookies will get
+    more secure than ever.
```

Comment:
For this sentence, I would say
```
By switching from `:no_protection` to `:lax` in `config/initializers/cookies.rb`,
cookies won't be sent by browsers in cross-site POST requests.
`:strict` will disable cookies from being sent in both cross-site GET and POST requests.
```
to provide a little more detail on this feature.